src/main/java/codeu/controller/ActivityServlet.java

```diff
@@ -0,0 +1,18 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet class responsible for user registration.
+ */
```

Comment:
I think it's a good practice to @author <your name> (<your email>) when you create a class for future reference.
So it'll be 
/** 
 * description of the class
 *
 * @author Kelvin (email@codeustudents.com)
 */